library/swarm

```diff
@@ -0,0 +1,4 @@
+# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
+# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
+
+latest: git://github.com/docker/swarm-library-image@c0f8204b2a1bb34233cd2144df51a4de9572440c
```

Comment:
On 4 February 2015 at 16:13, Andrea Luzzardi <notifications@github.com> wrote:
> We're not planning to push rc's in the library (at least, we don't think so unless we find a compelling reason).

I don't really see a compelling reason not to, especially if what
@vieux is telling me about you guys not doing official binaries
anymore is true (which also makes me kind of sad, but it's
understandable).